test/anchoredinterval.jl

```diff
@@ -197,24 +204,32 @@ using Intervals: Bounded, Ending, Beginning, canonicalize, isunbounded
         interval = AnchoredInterval{Day(1)}(startpoint)
         @test first(interval) == startpoint
         @test last(interval) == ZonedDateTime(2018, 11, 5, tz"America/Winnipeg")
+        @test minimum(interval) == startpoint
+        @test maximum(interval, increment=Hour(1)) == ZonedDateTime(2018, 11, 4, 23, tz"America/Winnipeg")
         @test span(interval) == Day(1)
 
         endpoint = ZonedDateTime(2020, 3, 9, 2, tz"America/Winnipeg")
         interval = AnchoredInterval{Day(-1)}(endpoint)
         @test_throws NonExistentTimeError first(interval)
+        @test_throws NonExistentTimeError minimum(interval, increment=Hour(1))
         @test last(interval) == endpoint
+        @test maximum(interval) == endpoint
```

Comment:
Be consistent with ordering
```suggestion
        @test_throws NonExistentTimeError first(interval)
        @test last(interval) == endpoint
        @test_throws NonExistentTimeError minimum(interval, increment=Millisecond(1))
        @test maximum(interval) == endpoint
```